src/tests/helpers.test.ts

```diff
@@ -0,0 +1,56 @@
+import { validateCountry, validateYear, validateInput, shortenPublicHoliday } from '../helpers';
+import { PublicHoliday } from '../types';
+
+describe('testing helpers', () => {
+  it('should enter valid country (validateCountry)', () => {
+    expect(validateCountry('GB')).toBeTruthy();
+  });
+  it('should enter invalid country (validateCountry)', () => {
+    expect(validateCountry('GG')).toBeFalsy();
+  });
+
+  it('should enter valid year (validateYear)', () => {
+    expect(validateYear(2023)).toBeTruthy();
```

Comment:
Will this test still work in 2024?

Reply:
I'd recommend to use some of date-mock packages (such as https://www.npmjs.com/package/mockdate or https://www.npmjs.com/package/jest-date-mock)

Reply:
It won't work in 2024)
Done.